release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -281,6 +281,18 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-node-observability-operator]
+==== Node Observability Operator (Technology Preview)
+{product-title} 4.11 introduces the Node Observability Operator in Technology Preview.
+
+Node Observability Operator provides the following features:
+
+* Deploy Node Observability Agents on the worker nodes.
+* Trigger a CRIO-O and Kubelet profiling.
+* Make the profiling data files available for the further analysis.
+
+For more information, see Understanding Node Observability Operator.
```

Comment:
xref to the Understanding Node Observability Operator content

Reply:
@ahardin-rh xref will be updated after https://github.com/openshift/openshift-docs/pull/47075 is merged.

Reply:
link updated.